proto/import_sstpb.proto

```diff
@@ -153,6 +153,10 @@ message CompactResponse {
 }
 
 message DownloadRequest {
+    // Map represents the map of <name, SSTMeta>.
+    // We'll generate all SSTMeta into one SST File.
+    map<string, SSTMeta> ssts = 1;
```

Comment:
is field id 1 never used before?

Reply:
It seems index starts from 2, and 1 was never used.